frontend/src/components/user/UnauthorizedMessage.tsx

```diff
@@ -0,0 +1,13 @@
+import { useTranslations } from "next-intl";
+import { Alert, GridContainer } from "@trussworks/react-uswds";
+
+export const UnauthorizedMessage = () => {
+  const t = useTranslations("Errors");
+  return (
+    <GridContainer className="margin-top-4">
+      <Alert type="error" heading={t("unauthorized")} headingLevel="h4">
+        {t("unauthorizedExplanation")}
```

Comment:
note that this is awaiting design input and will change later on - this is a placeholder for POC